### ISSUE TITLE
Syntax error at or near "group"

### ISSUE THREAD

**Issue by Set3007** [alerta/alerta#1447]
**Issue Summary**
Impossible to order by **group**

**Environment**
- OS: [Linux]
- API version: [8.3.3]
- Deployment: [Docker]

- Database: [Postgres v9.6.20]

- Server config:
  Auth enabled? Yes
  Auth provider? KEYCLOAK
  Customer views? Yes

- web UI version: [v8.4.0]

**To Reproduce**
1. Go to main page
2. Click on **group**
4. See error :

> Syntax error at or near "group" LINE 8: ORDER BY group ASC ^ (500)

 ```
syntax error at or near "group"
LINE 8:           ORDER BY group ASC
                            ^
```

**Expected behavior**
Expecting to order by group.

**Additional context**
I'm able to reproduce it playing with query, and it works with ORDER BY "group" (group with double quote).

**Comment by satterly** [comment]
Order what by group? You haven't given me much to go on. 😞

**Comment by satterly** [comment]
This issue is blocked waiting on more information. The description or subsequent comments do not provide enough information to triage, investigate or resolve the issue. Please review the description against the [issue template](https://raw.githubusercontent.com/alerta/.github/master/.github/ISSUE_TEMPLATE/bug_report.md) and ensure all relevant information is included. If you do not know what is expected you can ask on [Slack](https://slack.alerta.dev).

**Comment by satterly** [comment]
I'm going to assume this was fixed in change #1446 which was added after version 8.3.3, the version referred to here.
